examples/FiberProduct.g

```diff
@@ -13,8 +13,7 @@ N2 := FinSet( [ 2 .. 5 ] );
 #! <An object in FinSets>
 iota2 := EmbeddingOfFinSets( N2, M );
 #! <A monomorphism in FinSets>
-D := [ iota1, iota2 ];
-#! [ <A monomorphism in FinSets>, <A monomorphism in FinSets> ]
```

Comment:
This line includes information (being a mono) that might get lost in the future. It would be nice to retain this information.

Reply:
`iota1` and `iota2` are viewed in lines 10 and 14 which includes the "mono" information.